.env

```diff
@@ -1,6 +1,6 @@
 blits=true
+solid=true
+renderer=true
```

Comment:
You can leave it as is - planning to fix the sort by `mean` and not a static sort.